src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -431,15 +432,15 @@ public function handleError($type, $message, $file, $line, array $context)
     /**
      * Handles an exception by logging then forwarding it to an other handler.
      *
-     * @param \Exception|\BaseException $exception An exception to handle
-     * @param array                     $error     An array as returned by error_get_last()
+     * @param \Exception|\Throwable|\BaseException $exception An exception to handle
```

Comment:
IMO should be `@param \Throwable $exception An exception to handle` because Exception implements Throwable and BaseExeption no longer exists.